select_test.go

```diff
@@ -29,6 +29,30 @@ func TestSelectStmt(t *testing.T) {
 	require.Equal(t, 3, len(buf.Value()))
 }
 
+func TestSelectStmtWithSettings(t *testing.T) {
```

Comment:
Can we add a test case for the case where we call two `Settings`? i.e. `Select("a").From("table").Settings("", "").Settings("", "")`